content/200-orm/200-prisma-client/100-queries/030-crud.mdx

```diff
@@ -294,6 +294,58 @@ The following video demonstrates how to use `createMany()` and [faker.js](https:
 
 See [Working with relations > Nested writes](/orm/prisma-client/queries/relation-queries#nested-writes) for information about creating a record and one or more related records at the same time.
 
+### Create and return multiple records
+
+:::info
+
+This feature is available in Prisma ORM version 5.14.0 and later for PostgreSQL, CockroachDB and SQLite
+
+:::
+
+You can use `createManyAndReturn()` in order to create many records and return the resulting objects.
+
+<CodeWithResult outputResultText="query">
+<cmd>
+
+```ts
+const users = await prisma.user.createManyAndReturn({
+  data: [
+    { name: 'Alice', email: 'alice@prisma.io' },
+    { name: 'Bob', email: 'bob@prisma.io' },
+  ],
+})
+```
+
+</cmd>
+<cmdResult>
+
+```js no-copy
+[{
+  id: 22,
+  name: 'Alice',
+  email: 'alice@prisma.io',
+  profileViews: 0,
+  role: 'USER',
+  coinflips: []
+}, {
+  id: 23,
+  name: 'Bob',
+  email: 'bob@prisma.io',
+  profileViews: 0,
+  role: 'USER',
+  coinflips: []
+}]
+```
+
+</cmdResult>
+</CodeWithResult>
+
+:::warning
+
+`createManyAndReturn()` is not available with `relationLoadStrategy: join`.
```

Comment:
Why is that @Weakky?

Reply:
Isn't it the other way around anyway, that `relationLoadStrategy: join` is not available with `createManyAndReturn()`?
If so, why is this worth mentioning here? I could not find a similar mention for `createMany()` where I guess the same is also true?

Reply:
> Why is that @Weakky?
Because it's impossible to join relations on a `RETURNING` clause, so we have to send separate queries for every relation. But we don't have in-memory joining implemented for `relationLoadStrategy: join`.


> Isn't it the other way around anyway, that relationLoadStrategy: join is not available with createManyAndReturn()?
I'm not sure I understand the nuance between relationLoadStrategy not being available with createManyAndReturn or the other way around?

> I could not find a similar mention for createMany() where I guess the same is also true?
It is irrelevant for `createMany` because you cannot fetch relations anyway.

All in all, I don't mind removing this if you think it's confusing.

Reply:
It is missing an important point: only relations that have their foreign key present in the model that is being created can be fetched.

Reply:
> I'm not sure I understand the nuance between relationLoadStrategy not being available with createManyAndReturn and the other way around?

Let me expand:

> a) `createManyAndReturn()` is not available with `relationLoadStrategy: join`.

This implies that you can top level define `relationLoadStrategy: join` and then `createManyAndReturn()` will not be present in the API.

On the other hand:

> b) `relationLoadStrategy: join` is not available with `createManyAndReturn()`

This says that  `relationLoadStrategy: join` is not a valid option for the configuration inside a `createManyAndReturn()`.

If I understood correctly, you want to communicate b).

Reply:
> It is missing an important point: only relations that have their foreign key present in the model that is being created can be fetched.

@Weakky could I get some clarity here? Is this the case because it's not possible to create relations where the foreign key is not present in `createMany()` so that limitation is passed to `createManyAndReturn()`?

Reply:
Yes. It's not a `createMany` limitation per se, it's a database limitation.